src/wp-includes/class-wp-theme-json.php

```diff
@@ -1200,6 +1200,44 @@ public function get_custom_css() {
 		return $stylesheet;
 	}
 
+	/**
+	 * Returns the global styles base custom CSS.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return string The global styles base custom CSS.
+	 */
+	public function get_custom_base_css() {
+		return isset( $this->theme_json['styles']['css'] ) ? $this->theme_json['styles']['css'] : '';
+	}
+
+
+	/**
+	 * Returns the global styles per-block custom CSS.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return string The global styles per-block custom CSS.
+	 */
+	public function get_custom_block_css() {
+		$stylesheet = '';
+
+		// Add the global styles block CSS.
+		if ( isset( $this->theme_json['styles']['blocks'] ) ) {
+			foreach ( $this->theme_json['styles']['blocks'] as $name => $node ) {
+				$custom_block_css = isset( $this->theme_json['styles']['blocks'][ $name ]['css'] )
+					? $this->theme_json['styles']['blocks'][ $name ]['css']
+					: null;
+				if ( $custom_block_css ) {
+					$selector    = static::$blocks_metadata[ $name ]['selector'];
+					$stylesheet .= $this->process_blocks_custom_css( $custom_block_css, $selector );
+				}
+			}
+		}
+
+		return $stylesheet;
+	}
```

Comment:
The old [`get_custom_css`](https://github.com/WordPress/wordpress-develop/pull/5892/files#diff-e12c3008d747d1bec5be9927c5e7b1ced0a88641abe52c278d495da936714817R1183) function should be refactored to call these two to make it clear where the code came from.

Reply:
I'm going to pull these changes out and do them in a separate PR.